sashimi-webapp/src/logic/filemanager/history.js

```diff
@@ -0,0 +1,52 @@
+const previousHistory = [];
```

Comment:
The original intention was to make it a generic class solely for utilizing the stack. Will include a brief description on it.